### PR TITLE
fix postsubmit and imageusage issues of cluster-proxy-addon

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-release-2.5.yaml [hunk 1]
```diff
@@ -68,9 +68,11 @@ tests:
       cli: latest
       commands: |
         KUBECTL=oc
-        EXAMPLE_IMAGE_NAME=${IMAGE_FORMAT//\/stable:\$\{component\}//pipeline:cluster-proxy-addon}
-        export KUBECTL EXAMPLE_IMAGE_NAME
+        export KUBECTL
         make test-e2e
+      dependencies:
+      - env: EXAMPLE_IMAGE_NAME
+        name: cluster-proxy-addon
```
Comment (xuezhaojun) [inline ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-release-2.5.yaml:75]: @zhiweiyin318 I did as the https://github.com/openshift/release/blob/fee6200dbbdc15a0787674433f43834b3aa02795/ci-operator/config/open-cluster-management/multicloud-operators-foundation/open-cluster-management-multicloud-operators-foundation-release-2.4.yaml#L77 shows. But not sure and confident about what is going underlying. Do we need to add anything else?

File: ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-release-2.5.yaml [hunk 2]
```diff
@@ -101,6 +101,12 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ocm-ci-image-mirror
+- as: fast-forward
```
Comment (zhiweiyin318) [inline ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-release-2.5.yaml:104]: only main branch needs ff. release branch doesnot need ff. 
remove this.

Comment (xuezhaojun) [reply ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-release-2.5.yaml]: Done.

**Pull request merged**